packages/salesforce-adapter/src/metadata_deploy.ts

```diff
@@ -27,7 +27,8 @@ import { SaltoError,
   isContainerType,
   isAdditionChange,
   SaltoElementError,
-  SeverityLevel } from '@salto-io/adapter-api'
+  SeverityLevel,
+  ElemID } from '@salto-io/adapter-api'
```

Comment:
super nit: This will create a cleaner commit next time with only 1 line diffs
```suggestion
  SeverityLevel,
  ElemID,
} from '@salto-io/adapter-api'
```